src/main/reactjs/src/components/translator/TranslatorListing.tsx

```diff
@@ -9,13 +15,15 @@ import { ProgressIndicator } from 'components/elements/ProgressIndicator';
 import { TranslatorDetails } from 'interfaces/translator';
 import { Selectable } from 'interfaces/selectable';
 import { APIResponseStatus } from 'enums/api';
-import { useAppSelector } from 'configs/redux';
+import { useAppDispatch, useAppSelector } from 'configs/redux';
 import { useAppTranslation } from 'configs/i18n';
 import {
   addSelectedTranslator,
   removeSelectedTranslator,
 } from 'redux/actions/translatorDetails';
 import { publicTranslatorsSelector } from 'redux/selectors/translatorDetails';
+import { UiStates } from 'enums/app';
```

Comment:
Olisi hyvä kirjoittaa isoilla kirjaimilla kuten Airbnb:n best practicissa sanotaan --> acronyms and initialisms should always be all capitalized,